website/docs/user_guide/token.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: api-token
+title: How to create API Tokens
+---
+
+import newTokenImage from '/img/admin_create_token_form.png';
+
+In order to connect to Unleash clients will need an API token to grant access. A client SDK will need to token with "client privileges", which allows them to fetch feature toggle configuration and post usage metrics back.
+
+## Create API Token {#create-api-token}
+
+### Permissions {#permissions}
+
+<!-- TODO: update? -->
+
+To create an API token you'll need the CREATE_API_TOKEN permission level, at the time of writing (v4.0.0), this level is only set for instance admins.
+
+Eventually one should be able to customize which users has access to create tokens.
+
+All users are able to see tokens with CLIENT level access, but only instance admins can see tokens with ADMIN level access.
+
+### Step-by-step {#step-by-step}
+
+#### 1. Select `Configure` and `API Access` from the main menu {#step-1}
+
+This page is accessible from `/admin/api` url. ![API access in navigation](/img/admin_configure_api.png)
+
+<br />
+
+#### 2. Click `Add new API key` at the top right of the page {#step-2}
+
+If you already have tokens, you will be able to see keys in the table below. ![Add token](/img/admin_new_api_key_button.png)
+
+<br />
+
+#### 3. Fill and submit new token form {#step-3}
```

Comment:
If we keep this title maybe should be: "Fill and submit API token form"

Or: "Create API token options"